packages/govuk-frontend-review/src/common/middleware/vendor.mjs

```diff
@@ -4,13 +4,9 @@ import { packageNameToPath } from 'govuk-frontend-lib/names'
 const router = express.Router()
 
 /**
- * Add middleware to serve legacy code
+ * Add middleware to serve dependencies
  * from node_modules
  */
-router.use('/govuk_template/', express.static(packageNameToPath('govuk_template_jinja', 'assets')))
-router.use('/govuk_frontend_toolkit/assets', express.static(packageNameToPath('govuk_frontend_toolkit', 'images')))
-router.use('/govuk_frontend_toolkit/', express.static(packageNameToPath('govuk_frontend_toolkit', 'javascripts/govuk')))
 router.use('/iframe-resizer/', express.static(packageNameToPath('iframe-resizer', 'js')))
```

Comment:
Should we treat `govuk-frontend` as "vendor" too?

We currently have this line in [middleware/assets.mjs](https://github.com/alphagov/govuk-frontend/blob/255263fa125198199ccf26eebf6be1e4ab14e7cd/packages/govuk-frontend-review/src/common/middleware/assets.mjs):

```js
router.use('/assets', express.static(packageNameToPath('govuk-frontend', 'src/govuk/assets')))
```

Reply:
I can't say I have a strong preference, but it probably makes sense given we're now consuming Frontend as an essentially separate package.

Reply:
Well, that might be a reason. Seems that after moving the line between files, certain assets like the Transport font start 404-ing. Gonna keep them in-place for now, rather than dig into that.